source/agora/common/Config.d

```diff
@@ -1007,7 +1006,7 @@ public QuorumConfig toQuorumConfig (const ref SCPQuorumSet scp_quorum)
     import std.conv;
     import scpd.types.Stellar_types : NodeID;
 
-    Hash[] nodes;
+    ulong[] nodes;
```

Comment:
I'd like to remain as it is because there's some problem when sorting the node in Quorum.d.
```
quorum.nodes.sort;
```

Reply:
I tried it several ways. But there are a couple of things to handle after changing that. So I think it would be OK to remain as it is.
The problems:
1. Serializer/Deserializer should be changed.
2. New code is needed to sort the array of `NodeID`.